src/server/destination/exit-page/index.js

```diff
@@ -0,0 +1,6 @@
+import { Page } from '../../common/model/page/page-model.js'
+
+export class DestinationExitPage extends Page {
+  urlPath = '/destination/can-not-use-service'
```

Comment:
Is this right?  I was expecting 'cannot-user-service' (based on spelling)

Reply:
That is the URL in the prototype

Reply:
okay, let's go with that for now.  It looks like we'll be doing a url sweep later.